src/components/molecules/Dashboard/Workspace.tsx

```diff
@@ -11,6 +11,8 @@ import { Team as TeamType } from "@reearth/components/molecules/Dashboard/types"
 import { styled, useTheme, metrics } from "@reearth/theme";
 import { metricsSizes } from "@reearth/theme/metrics";
 
+import Avatar from "../Settings/Avatar";
```

Comment:
We have TS and webpack setup to resolve the `@reearth/` alias to `src/`. As you can see, MOST imports are written from the `@reearth` alias, except for components that are one time use and nested (see [here](https://github.com/reearth/reearth-web/blob/cfcd605a6c319d2b6a0b29e75a6690f92e7be9b5/src/components/molecules/Common/AssetModal/AssetContainer/index.tsx#L15-L17) as an example), so please write most imports from `@reearth/`.